scripts/collect_api_events.rb

```diff
@@ -22,8 +28,8 @@ def transform(body, pe_console, source_type)
       'sourcetype' => source_type,
       'event' => pe_event,
     }
-    events << ' ' unless events.empty?
-    events << event.to_json
+    events += ' ' unless events.empty?
+    events += event.to_json
```

Comment:
The `<<` operator here instead of `+=` would be better. It midifies the original string instead of creating a new one and replacing it in the background.

Reply:
Also, I bet you could get rid of the previous line if you made this one `events << "#{event.to_json} "` That would ensure you always have a space after each event, and my guess is the splunk endpoint doesn't care if there's a trailing space after the last event in the collection.

Reply:
Good idea